src/site.settings.php

```diff
@@ -1,16 +1,28 @@
 <?php
 
-use Platformsh\ConfigReader\Config;
+use Platformsh\ConfigReader\Config as PlatformshConfig;
 use wearewondrous\PshToolbelt\SiteSettings;
 
 $databases = [];
 $config_directories = [];
-$settings['container_yamls'][] = $app_root . '/' . $site_path . '/services.yml';
+
+$settings['container_yamls'][] = '../services/default.services.yml';
```

Comment:
Should we suport, multiple drupal versions, if this would change?

Reply:
We could do this with semver if we would require drupal core as dependency with a minimum version

Reply:
Use the `DRUPAL_VERSION` global to place version dependent includes there? For now, it is fine of course